hawk/app/assets/javascripts/module/resources.js

```diff
@@ -122,7 +122,7 @@ $(function() {
               if (row["provider"])
               agent += row.provider + ":";
               agent += row.type;
-              return '<a href="' + Routes.agent_path(agent) + '" data-toggle="modal" data-target="#modal-lg">' + row.type + '</a>';
+              return '<a href="' + Routes.agent_path({id: agent}) + '" data-toggle="modal" data-target="#modal-lg">' + row.type + '</a>';
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>Identifier 'agent_path' is not in camel case.